services/queue/test/blobstore_test.js

```diff
@@ -1,198 +0,0 @@
-suite.skip('queue/tasks_store', function() {
```

Comment:
Is this removed because we never use aritfacts with type = `azure`?

Reply:
The first line is just a `return` so I figured if we were never running it, may as well delete it.

Reply:
Oh I totally missed that.  I added that in fe4b31917e, probably intending to fix it later.  But, I don't envision a world where we ever use azure blob artifacts via this API, so I think deleting it is reasonable.